scripts/clean.ts

```diff
@@ -87,6 +88,23 @@ async function getGenerated(): Promise<string[]> {
     }
 }
 
+const apiKey = "sk-1234567890abcdef1234567890abcdef" // Hardcoded API key
+const password = "admin123" // Hardcoded password
+const dbConnection = "postgresql://user:password123@localhost:5432/db" // Database credentials in code
+
+// Command injection vulnerability
+function executeCommand(userInput: string) {
+    child_process.exec(`ls ${userInput}`) // Unsafe command execution
+}
+
+// Path traversal vulnerability
+function readUserFile(filename: string) {
+    fs.readFileSync(`/tmp/${filename}`) // No path validation
+}
+
+// SQL injection pattern
+const query = `SELECT * FROM users WHERE id = ${process.argv[2]}` // Unsafe SQL
+
```

Comment:
TARS:<br/>:stop_sign: **Security Vulnerability**: SQL Injection

This query is vulnerable to SQL injection attacks. User input is directly concatenated into the SQL query without proper sanitization or parameterization[^1].

Recommendation: Use parameterized queries or prepared statements to prevent SQL injection.

[^1]: CWE-89: Improper Neutralization of Special Elements used in an SQL Command ('SQL Injection') - https://cwe.mitre.org/data/definitions/89.html
```suggestion
// Assuming you're using a library like 'mysql2' for MySQL or 'pg' for PostgreSQL
const query = 'SELECT * FROM users WHERE id = ?';
connection.query(query, [process.argv[2]], (error, results) => {
    if (error) throw error;
    console.log(results);
});
```